codalab/server/bundle_manager.py

```diff
@@ -690,11 +690,10 @@ def _get_matched_workers(request_queue, workers):
         :param workers: a list of workers
         :return: a list of matched workers
         """
-        tagm = re.match('tag=(.+)', request_queue)
+        tag_match = re.match('tag=(.+)', request_queue)
         if tagm != None:
```

Comment:
If we change the `tagm` to `tag_match`, should this be changed as well?

Reply:
oops, yes I